blackbox/docs/sql/information_schema.txt

```diff
@@ -235,63 +235,67 @@ Columns Specification
-| ``collation_name``            | Not implemented                               |               |
+| ``collation_name``            | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``domain_catalog``            | Not implemented                               |               |
+| ``domain_catalog``            | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``domain_schema``             | Not implemented                               |               |
+| ``domain_schema``             | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``domain_name``               | Not implemented                               |               |
+| ``domain_name``               | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``user_defined_type_catalog`` | Not implemented                               |               |
+| ``user_defined_type_catalog`` | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``user_defined_type_schema``  | Not implemented                               |               |
+| ``user_defined_type_schema``  | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``user_defined_type_name``    | Not implemented                               |               |
+| ``user_defined_type_name``    | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``check_references``          | Not implemented                               |               |
+| ``check_references``          | Not implemented                               | ``String``    |
 +-------------------------------+-----------------------------------------------+---------------+
-| ``check_action``              | Not implemented                               |               |
+| ``check_action``              | Not implemented                               | ``Integer``   |
++-------------------------------+-----------------------------------------------+---------------+
+| ``generation_expression`      | Not implemented                               | ``String``    |
++-------------------------------+-----------------------------------------------+---------------+
+| ``is_generated``              | Not implemented                               | ``Boolean``   |
```

Comment:
`is_generated` is implemented - we implemented it already. We return whether columns is a `GeneratedReference` iirc.